### PR TITLE
[#2107] Checkboxes/radio buttons are now IE9 friendly

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2107-ie9-radio-buttons-have-no-selected-state-in-editor

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/76b430f92bffe39/output.txt

File: css/editors.less [hunk 1]
```diff
@@ -328,6 +328,10 @@
     margin-top: 2px;
   }
 
+  input[type="checkbox"]:checked {
+    box-shadow: 0 0 0 2px #3FB58E;
```
Comment (k88hudson) [inline css/editors.less]: lessify please:
`@green`

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/76b430f92bffe39/output.txt

Total script time: 1.65 mins

Server is running on http://pr959.butyr.org/templates/basic/
Run the Test Suite: http://pr959.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a781e0a2b9cd143/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/a781e0a2b9cd143/output.txt

Total script time: 0.92 mins

Server is running on http://pr959.butyr.org/templates/basic/
Run the Test Suite: http://pr959.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5d64588cfcdfcca/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5d64588cfcdfcca/output.txt

Total script time: 1.82 mins

Server is running on http://pr959.butyr.org/templates/basic/
Run the Test Suite: http://pr959.butyr.org/test/

**Pull request closed without merging**